### PR TITLE
Bfb/add arm component

### PR DESCRIPTION
**Pull request by brendanburkhart**

### CODE REVIEW

**Review by calebeby — DISMISSED** [review]

**Comment by brendanburkhart** [comment]
Is keeping the enum an inner class an improvement?

**Comment by calebeby** [comment]
I don't know I thought it was fine the way it was. Your choice.

**Comment by brendanburkhart** [comment]
Well it doesn't even work so...

**Comment by calebeby** [comment]
Oh, what?

**Review by calebeby — APPROVED** [review]

**Review by calebeby — CHANGES_REQUESTED** [review]
plz fix

File: components/arm.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+class Arm:
+    """Main manipulator arm"""
+
+    arm_motor = ctre.CANTalon
+    extended_limit_switch = wpilib.DigitalInput
+    retracted_limit_switch = wpilib.DigitalInput
+    direction = Direction.retract
+
+    def extend(self):
+        """Extend arm"""
```
Comment (calebeby) [inline components/arm.py]: Redundant

File: components/arm.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        self.direction = Direction.extend
+
+    def retract(self):
+        """Retract arm"""
```
Comment (calebeby) [inline components/arm.py]: Redundant

**Review by calebeby — CHANGES_REQUESTED** [review]
plz fix

File: components/arm.py (hunk 1 above)
Comment (calebeby) [reply components/arm.py]: Literally this is called as `Arm.extend()`

**Comment by calebeby** [comment]
Not dumb... just `helpful.not() # not helpful`

**Review by calebeby — APPROVED** [review]

**Pull request merged**